src/screens/InputUsername/index.js

```diff
@@ -289,13 +289,14 @@ const ChooseUsername = () => {
                     style={styles.image}
                   />
                   <View style={styles.icon}>
-                    <MemoOnboardingChangeProfilePlusIcon />
+                    <MemoOnboardingChangeProfilePlusIcon fill={COLORS.signed_primary} />
                   </View>
                 </View>
               </TouchableOpacity>
               <View style={{flex: 1}}>
                 <Input
                   placeholder="Username"
+                  placeholderTextColor={COLORS.gray400}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fill color for the `MemoOnboardingChangeProfilePlusIcon` component and the placeholder text color for the `Input` component have been updated. Verify these changes are consistent with the application's design guidelines.
```diff
- <MemoOnboardingChangeProfilePlusIcon />
+ <MemoOnboardingChangeProfilePlusIcon fill={COLORS.signed_primary} />
- placeholder="Username"
+ placeholderTextColor={COLORS.gray400}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->